fetch/SportZone.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from game import HockeyGame
+
+
+# Create game object for sportzone backed websites (KHL, Pride)
+def createSportZoneGame(cols, team):
+    # Create textual representation
+    cols_text = None
+    if ("Preview" in cols[-1].getText()):
+        opponent = cols[-4].getText()
+        cols_text = ", ".join(map(lambda e: e.getText(), cols[1:-4:]))
+        cols_text += ", ^" + team['name'] + " vs " + opponent
+    else:
+        opponent = cols.pop(5).getText()
+        cols_text = ", ".join(map(lambda e: e.getText(), cols[1:-1:]))
+        cols_text += ", ^" + team['name'] + " vs " + opponent
+    textual_rep = cols_text
+
+    # Find game time
+    dateText = cols[1].getText()
+    timeText = cols[2].getText()
+
+    ## Get day
+    day_ret = int(dateText.split(" ")[2])
+
+    ## Get month
+    month_text = dateText.split(" ")[1]
+    month_ret = None
+
+    match month_text:
+        case 'Apr':
```

Comment:
the only difference in this file and the original version in createKHLGame is this i added april